crossing-training.lic

```diff
@@ -679,18 +685,18 @@ class CrossingTraining
 
   def climb_branch
     start_script('skill-recorder') unless Script.running?('skill-recorder')
-    walk_to(2245)
+    walk_to @climbing_target['id']
     check_research
     if DRSkill.getxp('Athletics') < 29
-      walk_to(2245)
+      walk_to @climbing_target['id']
       if UserVars.athletics < 550
```

Comment:
I'm not really sure, I havent had a character have enough athletics use this part of the script yet